packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -140,12 +145,6 @@ label,
     &::after {
       opacity: 1;
     }
-
```

Comment:
Yay removed a calc! 🎉

Reply:
Oh, looks like Mgov has overrides for this style. Pushing up a fix addressing that too.

Reply:
Hey, it looks nice!